server/scripts/sendEmailBlast.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const axios = require('axios');
+const { dbClient } = require('../db/db');
+
+const CHES_HOST = process.env.CHES_HOST || 'https://ches-dev.apps.silver.devops.gov.bc.ca';
+const AUTH_URL =
+  process.env.AUTH_URL ||
+  'https://dev.oidc.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token';
+const failedSends = [];
+async function authenticateChes() {
+  const params = new URLSearchParams();
+  params.append('grant_type', 'client_credentials');
+
+  const config = {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    auth: { username: process.env.CHES_CLIENT_ID, password: process.env.CHES_CLIENT_SECRET }, // service client ID and secret
+  };
+  try {
+    const response = await axios.post(AUTH_URL, params, config);
+    return response.data.access_token;
+  } catch (error) {
+    console.log({ '@@@@': error });
```

Comment:
```suggestion
    console.error(error);
```

This could use a little cleaning up maybe